packages/api-v2/src/AuthProvider.tsx

```diff
@@ -115,11 +117,16 @@ const AuthProvider = ({ children, cookieTimeout }: AuthProviderProps) => {
                     });
             })
             .catch(() => {
-                setIsLoading(false);
-                setIsError(true);
+                if (isMounted) {
```

Comment:
might need update once @rostislav-deriv PR is merged as there are other changes to the same code ;-/

Reply:
hope its not gonna conflict too much

Reply:
yep agreed 👍 I believe it wouldn't cause conflicts as that PR does not touch the cleanup function but will resolve any if it does